de_templates/ingest/Ingest_SourceType_SourceName_DQ/data_factory/pipelines/ARM_IngestTemplate.json.jinja

```diff
@@ -0,0 +1,267 @@
+                        "typeProperties": {
+                            "pipeline": {
+                                "referenceName": "Get_Ingest_Config",
+                                "type": "PipelineReference"
+                            },
+                            "waitOnCompletion": true,
+                            "parameters": {
+                                "config_container": "config",
+                                "config_path": {
+                                    "value": "ingest/@{pipeline().parameters.data_source_name}/ingest_sources",
+                                    "type": "Expression"
+                                },
+                                "config_file": "ingest_config.json"
+                            }
+                        }
+                    },
+                    {
+                        "name": "For_Each_Ingest_Entity",
+                        "type": "ForEach",
+                        "dependsOn": [
+                            {
+                                "activity": "Get_Ingest_Config",
+                                "dependencyConditions": [
+                                    "Succeeded"
+                                ]
+                            }
+                        ],
+                        "userProperties": [],
+                        "typeProperties": {
+                            "items": {
+                                "value": "@activity('Get_Ingest_Config').output.pipelineReturnValue['config'][0]['ingest_entities']",
+                                "type": "Expression"
+                            },
+                            "isSequential": false,
+                            "activities": [
+                                {
+                                    "name": "{{ data_source_type | upper }}_to_ADLS",
+                                    "type": "Copy",
+                                    "dependsOn": [
+                                        {
+                                            "activity": "Generate_Ingest_Query",
+                                            "dependencyConditions": [
+                                                "Succeeded"
+                                            ]
+                                        }
+                                    ],
+                                    "policy": {
+                                        "timeout": "1:00:00",
+                                        "retry": 0,
+                                        "retryIntervalInSeconds": 30,
+                                        "secureOutput": false,
+                                        "secureInput": false
+                                    },
+                                    "userProperties": [],
+                                    "typeProperties": {
+                                        "source": {
+                                            "type": "AzureSqlSource",
```

Comment:
Think we should parameterise this. Then, I don't think anything in this pipeline will be Azure SQL specific (although further work will need to be done to make sure it can support a different source / database type).

Reply:
done